diesel_cli/README.md

```diff
@@ -4,6 +4,34 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli, by default, requires [`libpq`][postgres], [`sqlite`][sqlite]
+and [`mysql`][mysql], some of which may already be included on your system. See
+which dependencies are already present with:
+
+```sh
+which psql
+which sqlite3
+which mysql
+```
+
+Once these dependencies are installed, you can run `cargo install diesel_cli`.
+
+To install the cli without these dependencies, omit the unneeded dependencies from
+the following command:
+
+```
+cargo install diesel_cli --no-default-features --features postgres,sqlite,mysql
+```
+
+The argument passed to `--features` must include no spaces, and be comma-separated.
```

Comment:
```
546 ~/Documents/code/diesel > cargo install diesel_cli --force --features sqlite postgres --no-default-features
error: Invalid arguments.

Usage:
    cargo install [options] [<crate>]
    cargo install [options] --list
547 ~/Documents/code/diesel > cargo install diesel_cli --force --no-default-features --features sqlite postgres
error: Invalid arguments.

Usage:
    cargo install [options] [<crate>]
    cargo install [options] --list
```

Can you double check that the space separate version works for you? Maybe I have something weird on my machine